FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h

```diff
@@ -586,6 +592,40 @@ NS_SWIFT_NAME(Auth)
                    completion:(nullable void (^)(FIRAuthDataResult *_Nullable authResult,
                                                  NSError *_Nullable error))completion;
 
+#if TARGET_OS_IOS || TARGET_OS_TV || TARGET_OS_OSX || TARGET_OS_MACCATALYST
+/**
+ @fn startPasskeySignInWithCompletion:
+ @brief start sign in with passkey retrieving challenge from GCIP and create an assertion request.
+ @param completion Optionally; a block which creates a assertation request.
+
+ @remarks // TODO @liubinj add possible error codes
```

Comment:
this is a fast follow up after private preview.  Let me know if you think it's better to remove it for now @kevinthecheung